### PR TITLE
feat(hub): verify ethers signatures

### PR DESCRIPTION
**Pull request by Songkeys**

### CODE REVIEW

File: hub/pkg/util/nacl/verify_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want bool
+		err  error
+	}{
+		{
+			name: "valid",
```
Comment (Songkeys) [inline hub/pkg/util/nacl/verify_test.go:26]: I haven't passed this nacl test. I'd appreciate if someone could help me figure out what's wrong.

File: hub/pkg/util/nacl/verify_test.go [hunk 2]
```diff
@@ -26,8 +26,8 @@ func TestVerify(t *testing.T) {
...
-				sig:    "s1IkjK1hD+DzaMvtFSJ9Y9Y4/bkvzwynDlHPsfwYezpH6urnZfL1ZPaY3GZ6AlntfHVcFU5NkecoZ6KbyeBOBA==",
-				pubkey: "NItLuV6qn0hybnExRU4Q42/ISGqYMpI2VHmbRcPdka0=",
+				sig:    "Kz47gL88TIJPnHcNBld5dLe7P005AFPnN/haqSPPyufb6/ZXdiQNvZVHI0ImpWUhhMPRnh6tGIZnpX45wcMCBQ==",
+				pubkey: "MQ4rRFwbMa8BNsf8r+r/mCLT1qiZotWaayq//iMdbDc=",
```
Comment (Songkeys) [inline hub/pkg/util/nacl/verify_test.go:30]: why is this part changed? i scraped the data from mongodb directly. is it wrong? @Candinya

Comment (Candinya) [reply hub/pkg/util/nacl/verify_test.go:30]: > why is this part changed? i scraped the data from mongodb directly. is it wrong? @Candinya

yup, it cannot pass js tweetnacl library (maybe caused by `$oid` or something like that cause RSS3Index don't have them?)

check research repo at [nacl-test](https://github.com/NaturalSelectionLabs/nacl-test) (raw.js for this and index.js for success demo)
